ds-caselaw-ingester/lambda_function.py

```diff
@@ -38,23 +50,21 @@ def extract_lambda_versions(versions: List[Dict[str, str]]) -> List[Tuple[str, s
     return version_tuples
 
 
-def store_metadata(uri: str, metadata: Dict[str, Union[str, dict, List[dict]]]) -> None:
-    # Store Transformation Engine metadata
-    api_client.set_property(uri, name="transform-version", value=metadata["int-tre-version"])
-    for key, version in extract_lambda_versions(metadata["lambda-functions-version"]):
-        # int-te-bagit-checksum-validation, int-te-files-checksum-validation, and int-text-parser-version
-        api_client.set_property(uri, name=f'transform-{key}', value=version)
+def store_metadata(uri: str, metadata: dict) -> None:
+    tdr_metadata = metadata["parameters"]["TDR"]
+
```

Comment:
I think we need to store `transform-version`, `int-te-bagit-checksum-validation`, `int-te-files-checksum-validation`, and `int-text-parser-version` still? They look like they've gone...

The checksum ones aren't particularly important, but I'd like to store `transform-version` (the version of the transofrmation engine itself) and `transform-parser-version` (the version of the parser inside it, `int-text-parser-version` in the payload)

Reply:
They're not part of the new metadata format it seems, which is why they've been removed

Reply:
> For now the new structure does not include them, but we are planning to include them in the next release

... ok then, we can add it back then, I guess.